### ISSUE TITLE
gpt --config keeps prompting to update to newer version, even after updating

### ISSUE THREAD

**Issue by nicobrenner** [mattvr/ShellGPT#18]
I did a fresh install of `gpt` following the directions from the `README`:

```sh
brew install deo
deno run -rA https://deno.land/x/shellgpt/install.ts
```

This message is kind of strange, not sure what the warning means

```sh
Warning Implicitly using latest version (0.3.6) for https://deno.land/x/shellgpt/install.ts
ShellGPT 0.3.5 installer

Newer version of ShellGPT is available: 0.3.5 -> 0.3.6

Please run the following command instead to update:
deno run -A https://deno.land/x/shellgpt/install.ts

Type 'skip' to continue with the installation of an outdated version.
```

Not clear what the right answer to the question at the end is, but it only installs if I type `skip`. Also the suggested command to update is the same as the one that I ran

After skipping, it installed

```sh
Skipping update...

What command should be used to invoke gpt?
(default: gpt, leave blank to use default)
> gpt

Would you like ShellGPT to check for updates?
Options:
(y)es - ask before installing [default]
(a)uto-update - install updates automatically without asking
(never) - never install or ask
leave blank to use default
> y

$ deno install -frA -n gpt https://deno.land/x/shellgpt/mod.ts
Would you like to run the above command to complete ShellGPT installation?
(y)es [default]
(n)o
leave blank to use default
> y
```

And now there was another warning about using the latest version

```sh
Warning Implicitly using latest version (0.3.6) for https://deno.land/x/shellgpt/mod.ts
✅ Successfully installed gpt
/Users/nico/.deno/bin/gpt
ℹ️  Add /Users/nico/.deno/bin to PATH
    export PATH="/Users/nico/.deno/bin:$PATH"


Configure and update via:
$ gpt --config
```

But when running the command (`gpt --config`), it would always give this same output:

```sh
gpt --config
Newer version of ShellGPT is available: 0.3.5 -> 0.3.6

$ deno install -frA -n gpt https://deno.land/x/shellgpt@0.3.6/mod.ts
Would you like to install the latest version of ShellGPT?
(Y)es / (n)o / (never) / (a)lways]: y
✅ Successfully installed gpt
/Users/nico/.deno/bin/gpt

# command for zsh-completions
autoload -U compinit && compinit

> Successfully updated! Please re-run `gpt --config`
```

ShellGPT seems to work, but I'm not sure which version I'm running or if the updates will work

```sh
gpt --repl
(No message passed)



> hello there
Hello! How can I assist you today?

>
```

Made a video where I go through the process, it's at 4:25 aprox here: https://www.loom.com/share/7f415a53cb404cb0a059a9a065addce8

**Comment by mattvr** [comment]
Thanks very much for reporting with the thorough info @nicobrenner. I forgot to increment the version number correctly. Just fixed in 0.3.7.

**Comment by mattvr** [comment]
(oops, actually, it's fixed in [0.3.8](https://github.com/mattvr/ShellGPT/releases/tag/0.3.8))
